R/prep_exposures_survey.R

```diff
@@ -18,45 +18,52 @@
 #' @return data.frame
 prep_exposures_survey <- function(results_portfolio,
                                   peers_results_aggregated,
-                                  sector = c("coal", "oil and gas"),
+                                  sector = c("coal", "oil_and_gas"),
```

Comment:
I am okay with doing this, but just mentioning that the underscores are inconsistent with how lower case technologies are mapped in P4B. So we end up having different conventions again